rst_files/julia_by_example.rst

```diff
@@ -704,17 +700,16 @@ But best of all is to avoid writing code altogether
     β = 0.9     
     f(v) = p .+ β * v # broadcast the +
     sol = fixedpoint(f, [0.8])
-    println("Fixed point = $(sol.zero), and |f(x) - x| = $(norm(f(sol.zero) - sol.zero)) in "*
-    "$(sol.iterations) iterations")
+    println("Fixed point = $(sol.zero), and |f(x) - x| = $(norm(f(sol.zero) - sol.zero)) in $(sol.iterations) iterations")
 
 
 The ``fixedpoint`` function from the ``NLsolve.jl`` library implements the simple fixed-point iteration scheme above
 
-Since the ``NLsolve`` library only accepts vector based inputs, we needed to make the ``f(v)`` function broadcast on the ``+`` sign, and pass in the initial condition as a vector of length 1 with ``[0.8]]``
+Since the ``NLsolve`` library only accepts vector based inputs, we needed to make the ``f(v)`` function broadcast on the ``+`` sign, and pass in the initial condition as a vector of length 1 with ``[0.8]``
 
 While a key benefit of using a package is that the code is clearer, and the implementation is tested, by using an orthogonal library we also enable performance improvements
 
-In particular, we can use the ``Anderson acceleration`` with a memory of 5 iterations, by changing a setting
+In particular, we can use the **Anderson acceleration** with a memory of 5 iterations, by changing a setting
```

Comment:
Good point. I think we can just eliminate this, since we updated `NLsolve` to make this the default 

```suggestion
```